src/components/Reviews.jsx

```diff
@@ -4,27 +4,47 @@ import { Review } from "./Review";
 
 export function Reviews() {
   const [reviews, setReviews] = useState([]);
+  const [categories, setCategories] = useState([]);
+  const [chosenCategory, setChosenCategory] = useState("all");
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
+    let url = "https://be-nc-games.onrender.com/api/reviews";
+    if (chosenCategory !== "all") {
+      url += `?category=${chosenCategory}`;
+    }
+
     axios
-      .get("https://be-nc-games.onrender.com/api/reviews")
+      .get(url)
       .then((response) => {
         setReviews(response.data.reviews);
         setIsLoading(false);
+
+        return axios.get("https://be-nc-games.onrender.com/api/categories");
+      })
+      .then((response) => {
+        const fetchedCategories = response.data.categories;
+        const fetchedCategoySlugs = fetchedCategories.map((category) => category.slug);
+        const newCategories = ["all", ...fetchedCategoySlugs];
+        setCategories(newCategories);
```

Comment:
this should be extracted into a component dedicated to listing the categories in a drop down